lib/filesys.cpp

```diff
@@ -682,7 +687,7 @@ static int boinc_rename_aux(const char* old, const char* newf) {
     //
     int retval = rename(old, newf);
     if (retval) {
-        char buf[MAXPATHLEN+MAXPATHLEN];
+        char buf[MAXPATHLEN+MAXPATHLEN+1+7];
```

Comment:
Why is this `+1+7`? "mv", 2 spaces and 4 quotes are 8 bytes, shouldn't this be `+1+8`?

Reply:
Both our maths is a bit beyond me, I must admit. There was a time in my life when I kept adding 17 without counting - because [17 being the first/smallest/least random number](https://www.quora.com/Why-is-17-considered-the-least-random-number-in-the-world-of-computer-programming). So, we should add the possibility that it was just a typo.

Good ole memories aside, yes, 1+2+6 sounds about right.